### PR TITLE
AddMainContentExtractorReader

### PR DESCRIPTION
**Pull request by HawkClaws**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] New Loader/Tool
- [ ] Bug fix / Smaller change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [x] I have added a library.json file if a new loader/tool was added
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama_hub/web/main_content_extractor/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+loader = MainContentExtractorReader()
+documents = loader.load_data(urls=['https://google.com'])
+```
+
+## Examples
+
+### LlamaIndex
+
+```python
+from llama_index import GPTVectorStoreIndex, download_loader
```
Comment (EmanuelCampos) [inline llama_hub/web/main_content_extractor/README.md:27]: GPTVectorStoreIndex is deprecated, can you change to VectorStoreIndex?

**Review by EmanuelCampos — COMMENTED** [review]
Looks good, I did just one small comment

File: llama_hub/web/main_content_extractor/README.md (hunk 1 above)
Comment (HawkClaws) [reply llama_hub/web/main_content_extractor/README.md]: Thanks for pointing that out.
GPTVectorStoreIndex has been changed to VectorStoreIndex

**Review by EmanuelCampos — APPROVED** [review]
Awesome! thanks for the contribution

**Pull request merged**